docs/source/release_notes/release-notes.rst

```diff
@@ -42,14 +42,36 @@ develop
     *    - Type
          - Change
 
+    *    -
+         - 
+
+.. <<<<------------------------------------------------------------------------------------------------------------->>>>
+
+=======
+v0.39.0
+=======
+
+19 Apr 2017
+
+.. list-table::
+    :widths: 5 40
+    :header-rows: 1
+
+    *    - Type
+         - Change
+
     *    - |improved|
-         - Refactored ``AvailableTimestamps`` reducing overzealous synchronization. Giving out timestamps is no longer blocking on refreshing the timestamp bound if there are enough timestamps to give out with the current bound.
+         - Refactored ``AvailableTimestamps`` reducing overzealous synchronization.
+           Giving out timestamps is no longer blocking on refreshing the timestamp bound if there are enough timestamps to give out with the current bound.
+           This improves latency of timestamp requests under heavy load; we have seen an approximately 30 percent improvement on internal benchmarks.
            (`Pull Request <https://github.com/palantir/atlasdb/pull/1783>`__)
 
     *    - |improved|
          - The lock server now logs to ``SlowLockLogger`` logger if a request takes more than a given time (10 seconds by default) to be processed.
```

Comment:
Just chatted with Himangi, and it looks like logs to the SlowLockLogger writes to the service log at INFO for any lock held longer than 10 seconds. This isn't very clear from the existing release notes. The things I would want to know as an end user are "What is getting logged?" and "How/where do I find it?". So _locks longer than 10s_ are being logged, and they can be found _at INFO level in my service logs_. 

The line about DEBUG logging is a bit confusing because it sounds like I need to enable DEBUG logging in order to see the new logging.